src/Nri/Ui/SortableTable/V1.elm

```diff
@@ -0,0 +1,362 @@
+module Nri.Ui.SortableTable.V1 exposing
+    ( Column, Config, Sorter, State
+    , init, initDescending
+    , custom, string, view, viewLoading
+    , invariantSort, simpleSort, combineSorters
+    )
```

Comment:
Out of scope side note... but any idea what https://package.elm-lang.org/packages/NoRedInk/elm-sortable-table/latest/Table is?

Reply:
Uhhhhhhh I don't remember that! It looks like it's a fork of evancz/elm-sortable-table -- I wonder why we forked it..